src/Core/CommandBus/Parser/CommandDefinition.php

```diff
@@ -1,84 +0,0 @@
-<?php
-/**
- * Copyright since 2007 PrestaShop SA and Contributors
- * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
- *
- * NOTICE OF LICENSE
- *
- * This source file is subject to the Open Software License (OSL 3.0)
- * that is bundled with this package in the file LICENSE.md.
- * It is also available through the world-wide-web at this URL:
- * https://opensource.org/licenses/OSL-3.0
- * If you did not receive a copy of the license and are unable to
- * obtain it through the world-wide-web, please send an email
- * to license@prestashop.com so we can send you a copy immediately.
- *
- * DISCLAIMER
- *
- * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
- * versions in the future. If you wish to customize PrestaShop for your
- * needs please refer to https://devdocs.prestashop.com/ for more information.
- *
- * @author    PrestaShop SA and Contributors <contact@prestashop.com>
- * @copyright Since 2007 PrestaShop SA and Contributors
- * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- */
-
-namespace PrestaShop\PrestaShop\Core\CommandBus\Parser;
-
-/**
- * Transfers commands and queries definition data
- */
-class CommandDefinition
```

Comment:
Even if we use Messenger instead of tacticien this DTO is still relevant, we shouldn't remove this that is used by our command to list infos about CQRS command Instead this should remain but the command will probably need to be modified to be compatible with the new Messenger implementation

And the tricky part will be how to tell apart messenger handlers related to CQRS commands from other ones that may be used for other use cases in the future or by other frameworks

Reply:
I'm thinking of something to differentiate CQRS commands handlers:

![Capture d’écran 2023-06-08 à 12 45 32](https://github.com/PrestaShop/PrestaShop/assets/13801017/df6da1d1-e3f6-4cfe-bd16-8042767a7087)

Maybe some other option could be added with the tag? Or we could introduce a custom tag `messenger.cqrs_handler` that would act the same as `messenger.message_handler` but at least we'd know it's part of the CQRS scope